src/fetchcode/package.py

```diff
@@ -399,11 +376,11 @@ def get_package_info(cls, package_url):
 
         version = package_url.version
         if version and version in IPKG_RELEASES:
-            archive = IPKG_RELEASES[version]
+            archives = IPKG_RELEASES[version]
```

Comment:
Why is it renamed to 'archives' ? does it contain more than a single archive? And does all archives have a single field "url" and "date"

Reply:
Reverted this change is not required.